source/chapters/introduction.rst

```diff
@@ -27,6 +27,9 @@ is appropriate for the present moment and that will mix well into the
 currently playing track(s). To use headphone cueing, you need at least 2
 separate audio outputs, typically provided by a USB audio interface (also known
 as a “sound card”, although most of them are not shaped like cards anymore).
+The shortcut for headphone cueing in Deck1 is T while for Deck 2 the shortcut is Y. 
+The short cut is for English keyboard layout, settings may be different depending on
+your language settings.
```

Comment:
Repeating this everywhere a hotkey is documented is tedious. I'm sure there's a better way to do this. Can we make `shortcut` a glossary term and explain in the glossary that the shortcuts in the manual are only for English keyboards?

Reply:
I think this would be nice, I am on it now!

Reply:
yeah, printing this hint once at the top of the Keyboard chapter should suffice.
```suggestion
The shortcut for headphone cueing in Deck1 is :kbd:`T` while for Deck 2 the shortcut is :kbd:`Y`. 
```

Reply:
As stated in the current review, let's _not_ add an appendix paragraph but link to Controlling Mixxx > keyboard instead and mention the English lyout there.